adapters/oidc/js/src/main/resources/keycloak.js

```diff
@@ -438,15 +438,13 @@
                 baseUrl = kc.endpoints.authorize();
             }
 
-            var scope;
-            if (options && options.scope) {
-                if (options.scope.indexOf("openid") != -1) {
-                    scope = options.scope;
-                } else {
-                    scope = "openid " + options.scope;
-                }
-            } else {
+            var scope = options && options.scope ? options.scope : kc.scope;
```

Comment:
If the current options already has a scope configured use that, otherwise use the scope that is configured on the kc (keycloak object) directly, if present.